File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.BusinessLayer/OrdenPedido.cs
using LogisticStorage.DataLayer;
using LogisticStorage.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.BusinessLayer
{
    public class OrdenPedido
    {

        public static List<OrdenPedidoEntity> ObtenerMain()
        {
            OrdenPedidoDB DB = new OrdenPedidoDB();
            return DB.ObtenerMain();
        }
        public static List<OrdenPedidoEntity> ObtenerItem(Int32 EmpresaId)
        {
            OrdenPedidoDB DB = new OrdenPedidoDB();
            return DB.ObtenerItem(EmpresaId);
        }
        public static Int32 Registrar(OrdenPedidoEntity Ent)
        {
            OrdenPedidoDB DB = new OrdenPedidoDB();
            DB.Registrar(Ent);

            return Ent.OrdenPedidoId;
        }
    }
}

File: ProyectoMysql/server/BusinessLayer/OrdenPedido.py
from DataLayer.OrdenPedidoDB import *
from EntityLayer.OrdenPedidoEntity import *
from Utilidades.Conexion.configMysql import StartTransaction, EndTransaction, Restore


class OrdenPedido:
    def ObtenerMain()-> list[OrdenPedidoMainModel] :
        try:
            return OrdenPedidoDB.ObtenerMain()
        except Exception as e:
            print(e)
            
    def ObtenerItem(Id: int)-> list[OrdenPedidoSaveModel]:
        try:
            return OrdenPedidoDB.ObtenerItem(Id)
        except Exception as e:
            print(e)
            
    def Registrar(Ent: OrdenPedidoSaveModel) -> OrdenPedidoSaveModel:
        try:
            StartTransaction()
            Value = OrdenPedidoDB.RegistrarDB(Ent)
            EndTransaction()
            return Value
        except Exception as e:
            Restore()
            print(e)

    def ObtenerFiltroOCO()-> list[OrdenPedidoFiltroOCOModel]:
        try:
            return OrdenPedidoDB.ObtenerFiltroOCO()
        except Exception as e:
            print(e)
